docs/content/blog/2023-roadmap.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "Porter's 2023 Roadmap"
+description: "What's on the docket for Porter in 2023"
+date: "2023-01-31"
+authorname: "Carolyn Van Slyck"
+author: "@carolynvs"
+authorlink: "https://carolynvanslyck.com/"
+authorimage: "https://github.com/carolynvs.png"
+tags: ["roadmap"]
+---
+
+Now that 2022 is finally over and [Porter v1 is out the door](https://getporter.org/blog/v1-is-here/), I'd like to share our plans for this coming year.
+<!--more-->
+
+You can find a review of 2022 and our high level plans for 2023 in our [Porter Annual Review for 2022](https://github.com/cncf/toc/pull/951/files) but let's go over the highlights for a sneek peak of what's to come.
+
+* [Porter Operator](#porter-operator)
+* [Advanced Dependencies](#advanced-dependencies)
+* [Improved Bundle Security](#improved-bundle-security)
+* [Signing and SBOMs](#signing-and-sboms)
+
+### Porter Operator
+
+The [Porter Operator](/operator/) is a Kubernetes operator that runs bundles on your cluster. With the operator you can:
+
+* Automate running bundles on Kubernetes
+* Upgrade to new versions of bundles when they are released
+* Integrate bundle deployments into your existing pipelines
+
+Many users, especially adopters with large-scale Porter deployments, are eagerly awaiting managing their installations using a GitOps based workflow with the Porter Operator and Flux. We made great progress on the operator last year and our [v1 milestone](https://github.com/getporter/operator/milestone/1) outlines the remaining work necessary for a stable release.
```

Comment:
Do we want to encourage people to try it out and give feedback?

Reply:
Yes, absolutely. I'll add a call to action for that.